app/helpers/assignment_index_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module AssignmentIndexHelper
+  def cal_day(day:)
+    'current-day' if day == Date.current
+  end
+
+  def cal_assignment(assignment:, day:)
+    return if assignment.blank?
+    ass = 'assignment'
```

Comment:
I thought it would be okay in this case to abbreviate since I cant use assignment and it's only a local variable. But I can certainly change it to something else less, conspicuous?